editor/components/post-preview-button/index.js

```diff
@@ -38,11 +38,9 @@ export class PostPreviewButton extends Component {
 	 */
 	setPreviewWindowLink( url ) {
 		const { previewWindow } = this;
-		if ( ! previewWindow || previewWindow.location.href === url ) {
```

Comment:
Note: I found that `previewWindow.location.href` was not returning the value I expected: For the new window, it would return the URL of the _editor tab_, not the window. (Chrome 67.0.3396.99)